develop/dev-guide-third-party-support.md

```diff
@@ -1,9 +1,13 @@
 ---
-title: Third-Party Libraries Support Maintained by PingCAP
-summary: Learn about third-party libraries support maintained by PingCAP.
+title: Third-Party Tools Support by TiDB
+summary: Learn about third-party tools support by TiDB.
 ---
 
-# Third-Party Libraries Support Maintained by PingCAP
+# Third-Party Tools Support by TiDB
+
+> **Note:**
+>
+> This document only lists some common third-party tools supported by TiDB. Third-party tools that are not listed do not mean that they are not supported. PingCAP has no way of knowing whether they use features that TiDB does not support and thus compatibility cannot be guaranteed.
```

Comment:
```suggestion
> This document only lists common third-party tools supported by TiDB. Some other third-party tools are not listed, not because they are not supported, but because PingCAP is not sure whether they use features that are incompatible with TiDB.
```